### ISSUE TITLE
Open all match highlighted links in new tabs

### ISSUE THREAD

**Issue by a-raccoon** [brandon1024/find#333]
I'd like to request the option to bulk open new tabs for links that match a search term.

I have searched far and wide for a regex link opener extension, alas to no avail.

Thoughts?

**Comment by brandon1024** [comment]
We don't support the ability to bulk open new tabs, but we do have a keyboard shortcut that you can use to open a link under the focused occurrence. This functionality is documented in the README:


| Windows Shortcut                                   | macOS Shortcut                                     | Action                                                                 |
| :------------------------------------------------- | :------------------------------------------------- | :--------------------------------------------------------------------- |
| <kbd>CTRL</kbd>+<kbd>⇧</kbd>+<kbd>↵</kbd>          | <kbd>^</kbd>+<kbd>⇧</kbd>+<kbd>↵</kbd>             | Follow the first highlighted link in the current occurrence focus      |

**Comment by a-raccoon** [comment]
@brandon1024 indeed, I had noticed that feature. This is why I'm [still] hoping my request might become a viable added feature given that the functionality is already partially implemented.  *fingers crossed*
